content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Read replicas'
+metaTitle: 'Read replicas'
+metaDescription: 'Learn how to set up and use read replicas with Prisma Client'
+tocDepth: 3
+---
+
+<TopBlock>
+
+Read replicas enable you to distribute workloads across database replicas for high-traffic workloads. The [read replicas extension](https://github.com/prisma/extension-read-replicas), `@prisma/extension-read-replicas`, adds support for read-only database replicas to Prisma Client.
+
+The read replicas extension supports Prisma versions 5.2.0 and higher.
+
+</TopBlock>
+
+## Setup the read replicas extension
+
+Install the extension:
+
+```terminal
+npm install @prisma/extension-read-replicas
+```
+
+Initialize the extension by extending your Prisma Client instance and providing the extension a connection string that points to your read replica in the `url` option of the extension.
+
+<!-- prettier-ignore-start -->
```

Comment:
Why is this needed?

Reply:
Maintain the formatting of the code snippet below :)